src/searchForm/datePicker/datePickerFullScreen/index.tsx

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components'
+
+import DatePickerFullScreen from './DatePickerFullScreen'
+
+const StyledDatePickerFullScreen = styled(DatePickerFullScreen)`
+  & {
+    box-sizing: border-box;
```

Comment:
@grosbouddha Didn't you want to add this globally?

Reply:
There's no way to add it globally in the library, however this file is actually useless since I don't set any size, so maybe I should just remove it.